docs/before/account_olcf.md

```diff
@@ -0,0 +1,151 @@
+and is always RSA-token based, rather than password
+based.  The NCCS account allows you access to "moderate"
+NCCS systes, like Frontier, Andes, DTN and Marble.
+
+The reason for the difference is that OLCF maintains
+"open" systems completely isolated from "moderate"
+systems - separate networks, filesystems, login nodes,
+etc.  All this makes NCCS much more secure.
+
+Getting an XCAMS account is easy, but still won't
+allow you access to any OLCF systems until you complete
+the process in Step 1.
+
+## Step 1 - Get a computer account
+
+Only user groups (also known as project-IDs) can gain
+access to OLCF systems.  Thus, to access OLCF machines,
+you must either:
+
+1. Apply for membership (and access) to a project using the [my.olcf.ornl.gov][1]
+   user portal.
+   - Fill out the form to request access to the specific
+     project(s) you are collaborating on.
+   - Additional help and information is available in the [OLCF User Guide][2]
+
+2. Apply to create a new project allocation:
+   * [DD Project Application][3]
+     - small (~20,000 node-hours max)
+     - focused on demonstrating emerging science and software at 1000+ node scale
+     - rolling reviews
+   * [INCITE][4]
+     - large (~100,000 - 2M node-hours)
+     - high impact science open to the world
+     - deadlines around June of each year.
+   * [ALCC][5]
+     - large (~100,000 - 2M node-hours)
+     - focused on research in DOE science priority areas
+     - deadlines around Feb of each year.
+
+In practice, you still need to do step (1) after receiving
+an allocation.  After completing the new user form,
+you will be given instructions on how to proceed.
+
+The following general steps happen:
+
+1. You agree to the [system use policies][6].
+   * Link above includes the necessary acknowledgement statement for publications related to work done on OLCF resources.
+
+2. The project PI approves your request via the my.olcf portal.
+
+3. *If Accessing Moderate Systems*: You receive an RSA token via mail,
+   and follow directions from OLCF staff on activating your token.
+
+4. You should now be able to login to OLCF systems
+   (e.g. `ssh <XCAMS>@flux.op.ccs.ornl.gov` or `ssh <NCCSID>@frontier.olcf.ornl.gov`)
+
+Further information and more detailed instructions are
+available from the [OLCF Accounts and Projects Documentation][10].
+
+
+## Step 2 - Understand OLCF Systems
+
+Now that you have access to OLCF, here
+are a few things to know:
+
+OLCF system [Best Practices][7] explain that
+data in `$HOME` is served by NFS, continuously
+snap-shotted, limited in size, and read-only
+during jobs.
+
+The major directories where work should occur
+are on the high-performance filesystem.
+Different systems have different filesystems mounted:
+
+* `/lustre/wolf2/<project-id>` (ascent,odo)
+* `/lustre/polis/<project-id>` (defiant)
+* `/gpfs/alpine2/<project-id>` (summit)
+* `/lustre/orion/<project-id>` (frontier, andes)
+
+This is an NFS `/ccs/proj/<project-id>` directory
```

Comment:
NIT: I am not sure I follow here. Is this supposed to be reading "There is an NFS ..."?